apple-silicon-support/modules/default.nix

```diff
@@ -28,6 +28,12 @@
           }
         else
           pkgs;
+
+      warnings = lib.optional (lib.versionAtLeast pkgs.mesa.version "25.3") ''
```

Comment:
There really should be a way to silence this warning

Reply:
Thanks, good idea. Hopefully this won't stay around for long, if it does we may add our own pin.